data/core/macros/abilities.cfg

```
@@ -198,9 +198,9 @@ This ability will not cure an affected unit of poison, however, only delay its e
         cumulative=no
         name= _ "leadership"
         female_name= _ "female^leadership"
-        description= _ "This unit can lead your own units that are next to it, making them fight better.
+        description= _ "This unit can lead your troops in battle.
```

```suggestion
        description= _ "This unit can lead other troops in battle.
```

Needs to not have "your", enemy units can have leadership too (yes I know the original was not exactly right).